packages/twenty-front/src/modules/ui/input/button/components/FloatingButton.tsx

```diff
@@ -87,6 +94,7 @@ const StyledButton = styled.button<
   &:focus {
     outline: none;
   }
+  text-decoration: ${({ to }) => to && 'none'};
```

Comment:
text-decoration should actually always be 'none', let's remove 'to' from this condition and from the styledButton signature